NavigationReactNative/sample/twitter/App.js

```diff
@@ -26,7 +26,7 @@ stateNavigator.navigate('home');
 
 export default () => (
   Platform.OS === 'ios' ? (
-    <TabBarIOS>
+    <TabBarIOS barTintColor="#ffffff" tintColor="#006dbf">
```

Comment:
I changed the `tintColor` but it didn't seem to do anything. Is it meant to change the color of the tab titles? Don't we need the same approach as you took with the `NavigationBar` and set the `titleTextAttributes`?

Reply:
Whoops this should have been updated. I changed the name of the property to `selectedTintColor` to be more clear.